### ISSUE TITLE
Profile View : Clicking on *Profile* option in the Navigation Drawer leads to nothing

### ISSUE THREAD

**Issue by jig08** [WomenWhoCode/WomenWhoCode#327]
Clicking on **Profile** option in the Navigation Drawer leads to nothing.

**Comment by zassmin** [comment]
@jig08 yes, it needs design, but feel free to create something basic in the meantime!

**Comment by jig08** [comment]
@zassmin Sure, I shall :)

**Comment by zassmin** [comment]
@opticod this one would be a good one to tackle if you are interested. The existing profile layout can be reused for now. @jig08 are you working on it?

**Comment by anubhakushwaha** [comment]
@zassmin okay ma'am as you say,will start as soon as @jig08 gives me the approval for the same.Thanks :)

**Comment by jig08** [comment]
@zassmin Nope, I am not. 
@anubhakushwaha You can go ahead with this :)

**Comment by anubhakushwaha** [comment]
@zassmin any specifications or just display the data of the user which can be edited?

**Comment by zassmin** [comment]
@anubhakushwaha sorry for the delay (been traveling). Yes, that's it! Please reference/reuse the https://github.com/WomenWhoCode/WomenWhoCode/blob/master/app/src/main/res/layout/activity_userprofile.xml. It might make more sense to turn it into a fragment for reuse, so this could be a fun refactor PR. 😸
